src/command_modules/azure-cli-network/azure/cli/command_modules/network/dns_zone.py

```diff
@@ -22,3 +22,21 @@ def create_dns_zone(client, resource_group_name, zone_name, location='global', t
         kwargs['if_none_match'] = '*'
 
     return client.create_or_update(**kwargs)
+
+
+#pylint: disable=too-many-arguments
+def update_dns_zone(client, resource_group_name, zone_name, location='global', if_match=None,
+                    **operation_config):
```

Comment:
I don't know any custom commands where we expose `operation_config`. Does this actually appear in -h?